src/Lib/optionmm/command_line.hpp

```diff
@@ -213,7 +213,7 @@ struct toggle_value<bool>
         x = !x;
     }
 #if defined(__GNUC__) && __GNUC__ >= 2
-    inline void operator()(std::_Bit_reference x)
+    inline void operator()(std::vector<bool>::reference x)
```

Comment:
Nice! I was reading about this but didn't know the syntax. funny if a bit reference is just a list of bools in memory

Reply:
I had no idea... I was just taking a second look and saw this

../src/Lib/optionmm/command_line.hpp:211:17: note: candidate function not viable: no known conversion from 'std::vector<bool>::reference' (aka '__bit_reference<std::vector<bool>>') to 'bool &' for 1st argument
    inline void operator()(bool& x)

So I figured maybe I'll just copy that since it looks like the vector<bool> thing is what it wanted. :)